### PR TITLE
Add additional parameter "&strm=true" check.

### PR DESCRIPTION
**Pull request by vlmaksime**
KODI 18 for *.strm files use description from library and set "watched" status when "setResolvedUrl" returns only path.

### CODE REVIEW

File: resources/lib/youtube_plugin/kodion/impl/xbmc/xbmc_items.py [hunk 1]
```diff
@@ -13,26 +13,32 @@
 def to_play_item(context, play_item):
     context.log_debug('Converting PlayItem |%s|' % play_item.get_uri())
 
+    is_strm = str(context.get_param('strm', False)).lower() == 'true'
```
Comment (anxdpanic) [inline resources/lib/youtube_plugin/kodion/impl/xbmc/xbmc_items.py]: can you move this below major_version, and change this to 
```is_strm = str(context.get_param('strm', False)).lower() == 'true' and major_version >= 18``` 
so it's never partially applied in versions less than 18

**Comment by anxdpanic** [comment]
Thanks for the contribution, can you address the one comment, and then PR against the master branch instead of 6.x.x

**Comment by vlmaksime** [comment]
Done

**Comment by anxdpanic** [comment]
Thanks again 👍

**Pull request merged**